### ISSUE TITLE
[BUG] Import settings on Firefox Beta Android doesn't work

### ISSUE THREAD

**Issue by phly95** [darkreader/darkreader#3273]
On Firefox Beta Android, if you try to import Dark-Reader-Settings.json, the file browser opens, you select the file, and then nothing. The settings are exactly the way they were before the attempted import.

**Comment by B0pol** [comment]
Is it a browser bug?

**Comment by InsanityOnABun** [comment]
This is occurring in Firefox stable as well.

**Comment by phly95** [comment]
It is a bug in Firefox Fenix. When the issue was opened, Firefox Stable was still Fennec. Now that stable is also Fenix, the bug happens on all mainline Firefox builds afaik.

**Comment by phly95** [comment]
This particular bug does work in Fennec F-Droid, which uses what Firefox Stable used at the time this post was created. However another issue https://github.com/darkreader/darkreader/issues/3272 where the edit button on a static theme does nothing also has issues with Fennec Firefox builds.

**Comment by Gusted** [comment]
Yeah, it seems like there is some issues with requesting File Access. I need to spend some time on this issue as first it's hard to monitor and debug a phone that uses some TCP that `web-ext` doesn't accept and it could be a browser bug and than it's going to be relying on Mozilla.
